src/resolvers/userResolver.js

```diff
@@ -134,6 +134,62 @@ const userResolver = {
       );
     },
 
+    changeUsername: async(_, { userInfo }, { req }) => {
+      const { oldUsername, newUsername } = userInfo;
```

Comment:
I do not think we need to extract the `oldUsername` from the `userInfo`. That would require the user to enter their current username by themselves. We can extract the `oldUsername` from the `req` context. Check `getCurrentUser` query to see how that can be done.